### DISCUSSION TITLE
Is LGWebOSRemote still compatible with webOS23?

### DISCUSSION THREAD

**Discussion by alimbada** [klattimer/LGWebOSRemote#161] [General]
My TV keeps asking me to ugprade to webOS23. Has anyone tested LGWebOSRemote with the latest webOS for compatibility?

**Comment by klattimer** [comment]
bit defender just announced a series of vulnerabilities... update.

**Reply by alimbada** [reply]
Sorry, not sure what you mean. Are the vulnerabilities in webOS 22 or webOS 23? And how do those affect this project?
